subjects/reversesecondhalf/README.md

```diff
@@ -0,0 +1,42 @@
+# ReverseSecondHalf
+
+### Instructions
+Write a function `ReverseSecondHalf()` that takes a string as an argument and prints the second half reversed. If the length of the string is odd, round it up.
+
+- Prints the second half reversed followed by newline `\n`.
+- if the string is empty, return `Invalid Output`.
+- If the string's length equals one, return it, followed by newline `\n`.
+
+### Expected function
+
+```go
+func ReverseSecondHalf(str string) string {
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
```

Comment:
Missing a space in between "function" and ":"

Reply:
It's fixed.